README.md

```diff
@@ -8,7 +8,59 @@ Queries for text objects are also included which help you to navigate, select, a
 
 ## Installation
 
-- If you want ReScript Tree-sitter in NeoVim, refer to [`nvim-treesitter-rescript`](https://github.com/nkrkv/nvim-tree-sitter-rescript/) installation notes;
+### Neovim
+
+If you want ReScript Tree-sitter in NeoVim, you will first need to register a new parser for it like so:
+
+```lua
+local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
+parser_config.rescript = {
+  install_info = {
+    url = "https://github.com/nkrkv/tree-sitter-rescript",
+    branch = "main",
+    files = { "src/scanner.c" },
```

Comment:
I think `parser` and `scanner` should be added?
```suggestion
    files = { 'src/parser.c', 'src/scanner.c' },
```

Reply:
I think it's needed explicitly when this file is present, since it's not I don't think so. To support this I have my forked branch installed basically like this
```lua
  {
    "nvim-treesitter/nvim-treesitter",
    dependencies = {
      {
        "TheFedaikin/tree-sitter-rescript",
      },
    },
    opts = function(_, opts)
      vim.list_extend(opts.ensure_installed, {
        "rescript",
      })

      local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
      parser_config.rescript = {
        install_info = {
          url = "https://github.com/TheFedaikin/tree-sitter-rescript",
          branch = "main",
          files = { "src/scanner.c" },
          generate_requires_npm = false,
          requires_generate_from_grammar = true,
          use_makefile = true, -- macOS specific instruction
        },
      }
    end,
  },
```

And it works wonders